assets/l10n/app_en.arb

```diff
@@ -63,6 +63,18 @@
   "@actionSheetOptionUnstarMessage":  {
     "description": "Label for unstar button on action sheet."
   },
+  "errorAccountLoggedInTitle": "Account already logged in",
+  "@errorAccountLoggedInTitle": {
+    "description": "Error message when the account is already logged in."
+  },
+  "errorAccountLoggedIn": "Account logged in.\nServer: {server}\nUser: {email}",
```

Comment:
Here's a rewording:
```suggestion
  "errorAccountLoggedIn": "The account {email} at {url} is already in your list of accounts.",
```

One thing is that the terse headlinese version is already covered by the title; here in the body of the dialog box, we have a bit more room and can use complete sentences.